### PR TITLE
added resources to airflow components

### PR DESCRIPTION
**Pull request by stijndehaes**
#### What this PR does / why we need it:

It's add the ability to template resources to flower, airflow-web and airflow-scheduler

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by stijndehaes** [comment]
@gsemet thank you very much for reviewing a previous PR. This is another that I would like to get merged I resolved the conflicts just now. Could you have a look at this?

File: stable/airflow/README.md [hunk 1]
```diff
@@ -291,6 +291,9 @@ The following table lists the configurable parameters of the Airflow chart and t
 | `airflow.extraContainers`                | additional containers to run in the scheduler, worker & web pods | `[]`             |
 | `airflow.extraVolumeMounts`              | additional volumeMounts to the main container in scheduler, worker & web pods | `[]`|
 | `airflow.extraVolumes`                   | additional volumes for the scheduler, worker & web pods | `[]`                      |
+| `flower.resources`                       | enable workers                                          | `{}`                      |
```
Comment (gsemet) [inline stable/airflow/README.md]: Can you update the doc and add add resource for worker as well

**Comment by gsemet** [comment]
Minor change to do. Thanks for this PR

**Review by gsemet — CHANGES_REQUESTED** [review]

File: stable/airflow/README.md (hunk 1 above)
Comment (stijndehaes) [reply stable/airflow/README.md]: Updated the documentation, the workers already had resources added in the config :)

**Pull request merged**